### ISSUE TITLE
F2 and jump to definition etc not working

### ISSUE THREAD

**Issue by aathan** [juanfranblanco/vscode-solidity#404]
Sorry but I could not find a better place to ask this question:  Does this extension support F2 symbol renaming and command-click jump to definition / show places where the symbol under the cursor is utilized?  Those don't work for me but due to all the possible settings that could be affecting this I'm not sure if it's a settings problem or just not supported.

**Comment by juanfranblanco** [comment]
Yes it supports goto definition. What is your project setup?

**Comment by ink-victor** [comment]
> Renaming works for me via Cmd + F2.

True, although it's also true that in other languages `F2` itself opens up a text input where one can enter a new name then press `Enter` to apply it everywhere. This extension does not seem to support that or I too have a problem in my settings. 💭
